src/oatpp/web/server/HttpConnectionHandler.cpp

```diff
@@ -86,7 +87,21 @@ void HttpConnectionHandler::handleConnection(const std::shared_ptr<oatpp::data::
 }
 
 void HttpConnectionHandler::stop() {
```

Comment:
Once `stop()` was called, the `handleConnection()` should not populate any more threads.

Reply:
Oh yeah, will do!